entries/m/mail.163.com.json

```
@@ -0,0 +1,21 @@
+{
+  "NetEase Mail": {
+    "domain": "mail.163.com",
+    "additional-domains": [
+      "126.com",
+      "yeah.net"
+    ],
+    "tfa": [
+      "totp",
+      "sms"
+    ],
+    "documentation": "https://help.mail.163.com/faq.do?m=list&categoryID=297",
```

```suggestion
    "documentation": "https://help.mail.163.com/faq.do?m=list&categoryID=297",
    "notes": "SMS is required as a backup method",
```

Other sites that mandate SMS 2FA have a note